### PR TITLE
Blog post: Metal³: Kubernetes Native Bare Metal Cluster Management

### PR DESCRIPTION
**Pull request by alosadagrande**
This is a blog post related to the presentation given by Maël at Kubernetes and CNCF Finland meetup. It includes a link to the video talk and a brief explanation of the content that was explained.

Signed-off-by: Alberto Losada <alosadag@redhat.com>

This PR should close issue #124 

Thanks @maelk, I found the talk very interesting and concepts well explained.

### CODE REVIEW

File: _posts/2020-02-27-talk-kubernetes-finland-metal3.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+> The video recording from the “Kubernetes and CNFC Finland Meetup” is composed by three talks. The video embedded starts with Maël’s talk.
+
+> warning "Warning"
+> Playback of the video has been disabled by the author. Click on play button and then on "Watch this video on Youtube" link once it appears.
+
+<iframe width="1110" height="720" style="height: 500px" src="https://www.youtube.com/embed/3k5EfIQpw-E?t=4167" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<br>
+
+During the first part of the presentation, a detailed explanation of the different Kubernetes Custom Resource Definitions (CRDs) inside Metal³ and how they are linked with the Cluster API project. As an example, the image below shows the interaction between objects and controllers from both projects:
```
Comment (ptrnull) [inline _posts/2020-02-27-talk-kubernetes-finland-metal3.md]: a detailed explanation of the different Kubernetes Custom Resource Definitions (CRDs) inside Metal³ is shown and also how they are linked with the Cluster API project.

File: _posts/2020-02-27-talk-kubernetes-finland-metal3.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+<br>
+Next, Maël deeply explains each Custom Resource (CR) used during the provisioning of target Kubernetes clusters in bare metal servers. He refers to objects such as `cluster`, `BareMetalCluster`, `machine`, `BareMetalMachine`, `BareMetalHost` and so on. Each one is clarified with a YAML file definition of a real case and a workflow diagram that shows the reconciliation procedure.
+
+Last part of the talk is dedicated to executing a demo where Maël creates a *target Kubernetes cluster* from a running minikube VM (also called *bootstrap cluster*) where Metal³ is deployed. As it is pointed out in the video, the demo is running in *emulated hardware*. Actually, something similar to the [metal3-dev-env](https://github.com/metal3-io/metal3-dev-env) project which can be used to reproduce the demo. More information of the Metal³ development environment (metal3-dev-env) can be found in the [try-it](https://metal3.io/try-it.html) section. In case you want to go deeper, take a look at the blog post [A detailed walkthrough of the Metal³ development environment]({%post_url 2020-02-18-metal3-dev-env-install-deep-dive %}).
```
Comment (ptrnull) [inline _posts/2020-02-27-talk-kubernetes-finland-metal3.md]: Last part of the talk is dedicated to execute a demo where..
...be found in the [Metal³ try-it section](https://metal3.io/try-it.html).

File: _posts/2020-02-27-talk-kubernetes-finland-metal3.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+![metal3 introspection](/assets/2020-02-27-talk-kubernetes-finland-metal3/metal3-instrospection.resized.png)
+
+<br>
+The second part of the process is the **provisioning**. In this step, Maël explains how the Bare Metal Operator (BMO) is in charge along with Ironic to present the Operating System image to the physical server and complete its installation.
+
+![metal3 provisioning](/assets/2020-02-27-talk-kubernetes-finland-metal3/metal3-provisioning.resized.png)
+
+<br>
+Next, Maël deeply explains each Custom Resource (CR) used during the provisioning of target Kubernetes clusters in bare metal servers. He refers to objects such as `cluster`, `BareMetalCluster`, `machine`, `BareMetalMachine`, `BareMetalHost` and so on. Each one is clarified with a YAML file definition of a real case and a workflow diagram that shows the reconciliation procedure.
```
Comment (maelk) [inline _posts/2020-02-27-talk-kubernetes-finland-metal3.md]: Maybe "Cluster" and "Machine" to have the same capitalization ?

File: _posts/2020-02-27-talk-kubernetes-finland-metal3.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+At the end, the result is a new Kubernetes cluster up and running. The cluster is deployed on two emulated physical servers: one runs as the control-plane node and the other as a worker node.
+
+> info "Information"
+> The slides of the talk can be downloaded from [here](https://drive.google.com/open?id=1mdofzqIpH7XpFYkjB0ZC7EWU_RGW6aOl)
+
+## Speakers
+
+[Maël Kimmerlin](https://www.linkedin.com/in/maelkimmerlin/) Maël Kimmerlin is a Senior Software Enginner at Ericsson. In his own words:
```
Comment (maelk) [inline _posts/2020-02-27-talk-kubernetes-finland-metal3.md]: Engineer

**Pull request merged**